cluster/common.sh

```diff
@@ -942,9 +942,9 @@ EOF
 DISABLE_NETWORK_SERVICE_SUPPORT: $(yaml-quote ${DISABLE_NETWORK_SERVICE_SUPPORT})
 EOF
     fi
-    if [ -n "${DISABLE_ADMISSION_PLUGINS:-}" ]; then
```

Comment:
do you really want to get rid of DISABLE_ADMISSION_PLUGINS env var?

Reply:
Kubeup API server uses admission-control whereas local dev environment uses enable-admission-plugins/disable-admission-plugins. So the parameter "DISABLE_ADMISSION_PLUGINS" is removed because of conflicting with admission-control.

Reply:
will be handled by separate focused PR